lecture-notes/2-vcs.md

```diff
@@ -1758,22 +1758,114 @@ $
 If you had more remotes, they would also show up both in the configuration file
 and in the output of `git remote -v`.
 
-> As another aside, this convention of the path being `username/projectname` is
-> just that---convention. The server name and path name could be anything that
-> the server chooses. In GitHub's case, though, separating out by username
-> gives nice namespacing (same for the other forges).
+<!-- TODO(max): Figure out x-overflow -->
+<style>
+.sidenote {
+    font-size: 80%;
+    position: relative;
+    float: right;
+}
+/* Wide viewport */
+@media (min-width: 1400px) {
+    .sidenote {
+        clear: right;
+        margin-right: -23vw;
+        text-align: left;
+
+        top: -3rem;
+        width: 20vw;
+        margin-top: 1rem;
+    }
+}
+/* Narrow viewport */
+@media (max-width: 1400px) {
+    .sidenote {
+        text-align: right;
+        width: 100%;
+        margin: 1rem 0;
+        padding-left: 15%;
+    }
+}
+</style>
+
+<div class="sidenote" markdown=1>
+  As another aside, this convention of the path being `username/projectname` is
+  just that---convention. The server name and path name could be anything that
+  the server chooses. In GitHub's case, though, separating out by username
+  gives nice namespacing (same for the other forges).
+</div>
 
-So you have a remote now. You can tell Git to write a local ref to the remote
-by using the `push` subcommand:
+So you have a remote now. You can tell Git to write a local ref (in this case,
```

Comment:
"Write a local ref" feels a bit technical to me. Maybe say "copy the local commits from on a given tag or branch" instead?

Reply:
Trying slightly different wording:

> So you have a remote now. You can tell Git to write a local ref (in this case,
> the `main` branch and the commits it points to) to the remote (in this case,
> `origin`) by using the `push` subcommand:

Reply:
My concern here is that we don't say what "writing" a ref actually *does*, and I don't think the term alone makes it clear. I still think "copy" is a better verb.